src/common/containers/container.css

```diff
@@ -21,6 +21,7 @@
   padding: 2rem 0;
   position: relative;
   width: 100%;
+  border-bottom: 1px dotted $mid-grey;
```

Comment:
This would also add the border to `.lightStrip` (listed below), which possibly is not what we want (even if is hardly visible due to background colour).